fabfile/default_config.ini

```diff
@@ -0,0 +1,23 @@
+[insert_test]
```

Comment:
The format looks great 👍 

(A very minor suggestion `shard_counts_replication_factor ` > `shard_count_replication_factor `)